profiles/models.py

```diff
@@ -142,6 +180,8 @@ class Profile(models.Model):
     linkedin = JSONField(blank=True, null=True)
     student_id = models.IntegerField(blank=True, null=True, unique=True)
 
+    date_updated = models.DateTimeField(blank=True, null=True, auto_now=True)
```

Comment:
@gsidebo I'm proposing changing only `date_updated` -> `updated_on` and to deal with `date_joined_micromasters` -> `created_on` and `Model` -> `TimestampedModel` in a different issue since that will require some migration of existing data and backpopulating `updated_on` so that we can make the switch.